tests/test_collection_api.py

```
@@ -0,0 +1,113 @@
+"""Tests for Collection API endpoints."""
+
+import pytest
+from httpx import AsyncClient
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models import Collection, Thread
+
+
+@pytest.fixture
+async def sample_collection(auth_client: AsyncClient, async_db: AsyncSession) -> Collection:
```

Ruff (0.15.1)

[warning] Unused function argument: `auth_client`

(ARG001)